### DISCUSSION TITLE
Images with and without links are not vertically aligned properly

### DISCUSSION THREAD

**Discussion by jugmac00** [pradyunsg/furo#249] [Potential Issue] [answered]
For the tox4 documentation we list a couple of badges - while the most are linked and then get the `reference` class aligned, which cause a `vertical-align: middle`, for e.g. the implementation and the Python badge, a link makes not much of a sense, and so they do not get the `.reference` class.

As a result the badges are not properly aligned, see screenshot.

![Screenshot from 2021-09-16 13-38-44](https://user-images.githubusercontent.com/9895620/133605719-52c46e6c-ff68-4bce-8bbf-f78822175730.png)

Is there anything we could do on our side?

**Comment by pradyunsg** [comment, marked as answer]
Neato. Fixed in 0d97b66e which I'll release over the weekend / next week. :)

**Reply by jugmac00** [reply]
Awesome 👍
